### ISSUE TITLE
Oasis Biome cannot be generated

### ISSUE THREAD

**Issue by YaoKnowSnow** [TerraformersMC/Terrestria#309]
oasis biome cannot be generated.
This my fifth saves. Therefore, it is not too far away to can not be found.

![cannot_find](https://github.com/TerraformersMC/Terrestria/assets/138879711/889ea22e-874c-43dc-9653-1e57229076b0)

 <sub> I am come from China, I am not very good at English. So, if there are some grammatical errors here, please forgive me. Thanks. </sub>

**Comment by gniftygnome** [comment]
You are correct; the Oasis biome does not generate since 1.18.2.  We have left the biome defined so people can upgrade old games and still have the correct biome name and colors.  It would be nice to make the Oasis biome generate again in the future, but it requires some work (or we might make it a structure instead).
